### PR TITLE
Final commit for design related changes as referred to in #16.

### PR DESCRIPTION
**Pull request by akshat157**
Deleted obsolete files and folders. added template.html. Minor fixes/improvements.

Site preview - akshat157.github.io/STAC-IITMandi.github.io
template webpage preview - akshat157.github.io/STAC-IITMandi.github.io/template.html

### CODE REVIEW

**Comment by shreyasbapat** [comment]
What comes on the banner of the template?

File: template.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    <!-- particles.js container -->
+    <div id="particles-js"></div>
+
+  </div>
+  <!-- End Top Background Image Wrapper -->
+
+
+  <!-- Content comes here -->
+  <!-- Sample Content -->
+  <div class="wrapper row2">
```
Comment (shreyasbapat) [inline template.html]: Alleviated! Thanks!

**Comment by akshat157** [comment]
Gave a thought to the same. ended up with "It'd be nice to let it be up to the one who makes the pages." :)

**My suggestions** -

- "SPACE TECHNOLOGY AND ASTRONOMY CELL, IIT MANDI" written. left aligned.
- Page title comes there, instead of being in the content.

**Comment by shreyasbapat** [comment]
True! Merging

**Pull request merged**